contrib/opencensus-ext-logging/README.rst

```diff
@@ -0,0 +1,41 @@
+OpenCensus logging Integration
+============================================================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opencensus-ext-logging.svg
+   :target: https://pypi.org/project/opencensus-ext-logging/
+
+Installation
+------------
+
+::
+
+    pip install opencensus-ext-logging
+
+Usage
+-----
+
+.. code:: python
+
+    import logging
+
+    from opencensus.trace import config_integration
+    from opencensus.trace.samplers import AlwaysOffSampler
+    from opencensus.trace.tracer import Tracer
+
+    config_integration.trace_integrations(['logging'])
+    logging.basicConfig(format='%(asctime)-15s traceId=%(traceId)s spanId=%(spanId)s %(message)s')
```

Comment:
Why -15s?

Reply:
Trying to follow the sample code in https://docs.python.org/3.1/library/logging.html#logging.debug.
I guess the reason is to align with some Linux console tools.

Reply:
That's surprising to me since the other fields aren't padded, but feel free to leave it in if it looks standard to you.

Reply:
Given it is not the main purpose of the sample code, I've removed the `-15s` to avoid the confusion.